apps/web/components/settings/TravelScheduleModal.tsx

```diff
@@ -99,10 +99,13 @@ const TravelScheduleModal = ({
             <>
               <Label className="mt-2">{t("time_range")}</Label>
               <DateRangePicker
-                startDate={startDate}
-                endDate={endDate ?? startDate}
+                dates={{
+                  startDate,
+                  endDate: endDate ?? startDate,
+                }}
                 onDatesChange={({ startDate: newStartDate, endDate: newEndDate }) => {
-                  setStartDate(newStartDate);
+                  // If newStartDate does become undefined - we resort back to to-todays date
```

Comment:
@CarinaWolli can you test this all works as you expect - (i tested locally and it seems to be fine) 

undefined doesnt ever happen so this shouldnt be an issue